STAGES.md

```diff
@@ -57,10 +58,9 @@ to a particular way of solving a problem.
 
 > “This idea is becoming part of the web.”
 
-A **Candidate Recommendation** [championed] by a [W3C Working Group], usually
-implemented by at least 2 [recognized browser vendors], possibly behind a flag. |
-It should be considered stable and subject to little change. Stage 3 features
-will likely become a standard.
+A **Candidate Recommendation** [championed] by a [W3C Working Group] or implemented by at least 3 [recognized browser vendors], possibly behind a flag. It should be considered stable and subject to little change. Stage 3 features
```

Comment:
```suggestion
A **Candidate Recommendation** [championed] by a [W3C Working Group] or implemented by at least 3 [recognized browser vendors]. It should be considered stable and subject to little change. Stage 3 features
```

If it's behind a flag it should not be considered implemented in that browser for our purposes.

MDN uses this logic too.

Reply:
Nice catch